k8s/knative-operator/1.22/operator-1.7.2.yaml

```diff
@@ -17,13 +206,13 @@ kind: CustomResourceDefinition
 metadata:
   name: knativeeventings.operator.knative.dev
   labels:
-    operator.knative.dev/release: 'v1.2.2'
-    app.kubernetes.io/version: '1.2.2'
+    operator.knative.dev/release: 'v1.7.2'
+    app.kubernetes.io/version: '1.7.2'
     app.kubernetes.io/part-of: knative-operator
 spec:
   group: operator.knative.dev
   versions:
-    - name: v1alpha1
+    - name: v1beta1
```

Comment:
I see that CustomResourceDefinition has been updated.
What would be the upgrade path. 
Is this CRD deployed by helm or by kubectl apply ?

Reply:
That is the operator, which will update itself. But yes, if helm 3 can't update crds I guess we have to add a line to `upgrades.yaml`. Always

Reply:
knative has no crds folder and our skeleton chart overwrites them, which is safe since knative does the same